testing/references/bestest_hydronic_heat_pump/plot_references.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python2
```

Comment:
Same comment as with the `bestest_hydronic` case

Reply:
Thanks for the catch.  Yes, didn't mean to include.

Reply:
Done in https://github.com/ibpsa/project1-boptest/pull/314/commits/d8851fb75af5911cfdf2d089e8235238b2f71ac0.